src/plcc.py

```diff
@@ -113,9 +112,13 @@ def main():
     sem(nxt)    # semantic actions
 
 def plccInit():
-    global flags, STD, STDT, STDP
+    global flags, argv, STD, STDT, STDP
     STDT = ['ILazy','IMatch','IScan','ITrace', 'Trace', 'PLCCException', 'Scan']
     STDP = ['ProcessFiles','Parse','Rep']
+    print(argv)
+    if len(argv) > 1:
+        if argv[0] == "--python":
```

Comment:
plcc.py has a mechanism for handling command-line arguments. Unless there is a good reason, we should probably work within that framework.

Specifically, there is a loop in main() that walks through all the command line arguments and calls processFlag() to populate a global dict with flags. The rest of the system interrogates flags to decide what to do.